app/serializers/export/user_serializer.rb

```diff
@@ -17,8 +17,13 @@ class UserSerializer < ActiveModel::Serializer
     has_many   :comments, each_serializer: Export::CommentSerializer
 
     def comments
-      object.person.comments
+      Comment
```

Comment:
Why should we export of other users if they've been made to the export users posts?